tests/src/EventPageTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Drupal\Tests\localgov_events\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests LocalGov Event page.
+ *
+ * @group localgov_campaigns
+ */
+class EventPageTest extends BrowserTestBase {
+
+  /**
+   * Test breadcrumbs in the Standard profile.
+   *
+   * @var string
+   */
+  protected $profile = 'standard';
```

Comment:
The "standard" profile will lead to a longer test run.  This is something to keep in mind.

Reply:
:+1: Have updated it to use the 'minimal' profile instead.

Reply:
Thanks Steve.  I am assuming the default [*testing*](https://git.drupalcode.org/project/drupal/-/blob/9.0.x/core/tests/Drupal/Tests/BrowserTestBase.php#L117) profile is not adequate in this case which is why you are explicitly declaring the profile.  Sorry, I should have mentioned this last week :(

Reply:
Also, this test file is in the *tests/src/* directory rather than the more common *tests/src/Functional/* directory.  The namespace of the test class is also saying *Drupal\Tests\localgov_events\Functional*.  If you please double check this.